applications/minotari_miner/src/config.rs

```diff
@@ -110,6 +113,7 @@ impl Default for MinerConfig {
             wait_timeout_on_error: 10,
             wallet_payment_address: TariAddress::default().to_hex(),
             stealth_payment: true,
+            range_proof_type: RangeProofType::BulletProofPlus,
```

Comment:
can we make the default revealed.

Reply:
Let's just get @stringhandler's opinion as well